Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:alpine AS build-env
+FROM golang:alpine3.17 AS build-env
```

Comment:
Can you please squash this commit with the previous one, so that the total diff of the PR is minimal? It is a nit, but still a good practice.

Reply:
Squashing completed...